irods_browser_app/static/css/site_base_irods.css

```diff
@@ -8,10 +8,6 @@
     opacity: 0.7;
 }
 
-#irodsContent .modal-lg {
-    width: 570px;
-}
-
```

Comment:
@mauriel133 this ```modal-lg``` class is still used in irods_upload_create.html and irods_upload_add.html. Is is replaced by some other class definition? If not, I believe it needs to be kept since it is still used.